study-builder/studies/rarex/i18n/en.json

```diff
@@ -247,13 +247,13 @@
   "general_information_hic_not_to_answer_details": "What type(s) of coverage do you have?",
   "general_information_hic_not_to_answer_details_exp": "Please describe the general type of insurance(s) you have.",
 
-  "general_information_patient_country_birthplace": "Country/Territory where the patient was born*",
+  "general_information_patient_country_birthplace": "Country/Territory where the patient was born *",
   "general_information_patient_country_birthplace_req_hint": "Country/Territory cannot be empty",
 
-  "general_information_patient_city_birthplace": "Name of the city/municipality where the patient was born*",
+  "general_information_patient_city_birthplace": "City/Municipality where the patient was born *",
   "general_information_patient_city_birthplace_req_hint": "City/Municipality cannot be empty",
 
-  "general_information_patient_state_birthplace": "State where the patient was born*",
+  "general_information_patient_state_birthplace": "State/Province where the patient was born (if State/Province is not applicable to the birthplace, enter any other geographic descriptor (e.g. Republic, etc.) *",
```

Comment:
I know we're just copy-pasting whatever is provided to us in the ticket, but maybe we should add a missing parenthesis at the end to balance things out?
```
(if ... (e.g. ... etc)
                      ^- here
```

Reply:
Got rid of second opening parentheses. I believe this makes is more correct in English and Math.